docs/src/index.js

```diff
@@ -2,7 +2,7 @@ import React, { Fragment, useState } from "react";
 import { render } from "react-dom";
 
 // TODO: Fix workspace link
-import { Button, Grid } from "../../packages/web";
+import { Button, Grid, Base } from "../../packages/web";
```

Comment:
What `Base` does?

Reply:
Base is a Styled component that gets all of the helpers from `styled-system` applied to it ... My thought was I want there to be constraints on most elements on their api (what they can and can't do) but to have one generic component that can do everything